app/src/main/java/com/google/samples/apps/nowinandroid/MainActivity.kt

```diff
@@ -95,12 +98,16 @@ class MainActivity : ComponentActivity() {
         WindowCompat.setDecorFitsSystemWindows(window, false)
 
         setContent {
-            val systemUiController = rememberSystemUiController()
             val darkTheme = shouldUseDarkTheme(uiState)
```

Comment:
Not your code but both the variable `darkTheme` and the method `shouldUseDarkTheme` are poorly named and make reading this PR difficult. 

The variable which contains the user's dark theme preference should be: `shouldUseDarkTheme` or `darkThemePreference`, and the method which obtains the value should be `getDarkThemePreference`. 

If you could address in this PR that'd be great.